tt_metal/hw/inc/grayskull/eth_l1_address_map.h

```diff
@@ -16,14 +16,18 @@ struct address_map {
   static constexpr std::int32_t FIRMWARE_BASE = 0;
   static constexpr std::int32_t ERISC_MEM_MAILBOX_BASE = 0;
 
+  static constexpr std::int32_t ERISC_APP_ROUTING_INFO_SIZE = 0;
   static constexpr std::int32_t ERISC_APP_SYNC_INFO_SIZE = 0;
+  static constexpr std::int32_t ERISC_APP_ROUTING_INFO_BASE = 0;
   static constexpr std::int32_t ERISC_APP_SYNC_INFO_BASE = 0;
   static constexpr std::int32_t ERISC_L1_ARG_BASE = 0;
 
   static constexpr std::int32_t ERISC_FIRMWARE_SIZE = 16;
   static constexpr std::int32_t ERISC_APP_RESERVED_BASE = 0;
   static constexpr std::int32_t ERISC_APP_RESERVED_SIZE = 16;
   static constexpr std::int32_t ERISC_L1_UNRESERVED_BASE = 0;
+  static constexpr std::uint32_t SEMAPHORE_BASE = 0;
+  static constexpr std::uint32_t CQ_CONSUMER_CB_BASE = 0;
```

Comment:
why 0s?

Reply:
To fix GS builds, since eth FW is built by default for all arches. I guess deleting this file for GS can be another cleanup item?